test/e2e/test_manager.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+shopt -s expand_aliases
+
+# Entrypoint for all our end-to-end tests
+# @param $1 - Number of the test to run
+run_test() {
+  if [ -z "$1" ]; then
+    echo "Usage: ./test/e2e/run_test.sh [NUMBER]"
+    exit 1
+  fi
+  echo "🧠 Starting test manager, for test $1"
+  local TEST="$(ls -h test/e2e/ | grep $1)"
```

Comment:
needs to look at the number with a separator, otherwise we can have at most 9 tests...

Reply:
Good eye, let me fix that quickly.